fullstop-plugins/fullstop-snapshot-source-plugin/src/main/java/org/zalando/stups/fullstop/plugin/snapshot/SnapshotSourcePlugin.java

```diff
@@ -62,14 +65,14 @@ public SnapshotSourcePlugin(final UserDataProvider userDataProvider, final Viola
     @Override
     public void processEvent(CloudTrailEvent event) {
         List<String> instanceIds = getInstanceIds(event);
+        Map<String, String> metaData = newHashMap();
```

Comment:
maybe better to create it custom for every violation. to avoid to have wrong metadata information.